src/gsy_e/gsy_e_core/area_serializer.py

```diff
@@ -31,41 +33,46 @@
 from gsy_e.models.strategy.finite_power_plant import FinitePowerPlant # NOQA
 
 from gsy_e.models.leaves import Leaf # NOQA
-from gsy_e.models.leaves import *  # NOQA
-from gsy_framework.utils import convert_pendulum_to_str_in_dict, key_in_dict_and_not_none
+from gsy_e.models.leaves import *  # NOQA  pylint: disable=wildcard-import, unused-wildcard-import
 
 
 class AreaEncoder(json.JSONEncoder):
-    def default(self, obj):
-        if type(obj) is Area:
-            return self._encode_area(obj)
-        elif isinstance(obj, Leaf):
-            return self._encode_leaf(obj)
-        elif isinstance(obj, BaseStrategy):
-            return self._encode_subobject(obj)
-
-    def _encode_area(self, area):
+    """Json encoder for Area object."""
+
+    def default(self, o):
```

Comment:
Isn't pylint complaining about the one-char argument ?

Reply:
It was complaining about "Parameter 'o' has been renamed to 'obj' in overridden 'AreaEncoder.default' method". That's why I changed it.